docs/common-patterns.rst

```diff
@@ -86,9 +86,7 @@ This is as opposed to the more intuitive sending pattern:
                 // Check if call succeeds to prevent an attacker
```

Comment:
This comment is not relevant anymore. If you are not confident with making the relevant changes, then please just keep send and we will go over it in another iteration.